src/main/java/fi/okm/jod/yksilo/entity/Yksilo.java

```diff
@@ -37,16 +28,14 @@
 
 @Entity
 @Getter
-public class Yksilo {
-
+public class Yksilo extends JodEntity {
   @Id
-  @Column(updatable = false, nullable = false)
+  @Column(name = "id")
```

Comment:
name is unnecessary and the updatable false, nullable false are there on purpose